#
# These requirements were autogenerated by pipenv
# To regenerate from the project's Pipfile, run:
#
#    pipenv lock --requirements
#

-i https://pypi.org/simple
cachetools==4.2.2; python_version ~= '3.5'
certifi==2021.5.30
cffi==1.14.6
charset-normalizer==2.0.4; python_version >= '3'
google-api-core[grpc]==2.0.0; python_version >= '3.6'
google-auth==2.0.1; python_version >= '3.6'
google-cloud-bigquery==2.25.0
google-cloud-core==2.0.0; python_version >= '3.6'
google-crc32c==1.1.2; python_version >= '3.6'
google-resumable-media==2.0.0; python_version >= '3.6'
googleapis-common-protos==1.53.0; python_version >= '3.6'
grpcio==1.39.0
idna==3.2; python_version >= '3'
lxml==4.6.3; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
numpy==1.21.2; python_version < '3.11' and python_version >= '3.7'
packaging==21.0; python_version >= '3.6'
pandas==1.3.2; python_full_version >= '3.7.1'
proto-plus==1.19.0; python_version >= '3.6'
protobuf==3.17.3
pyasn1-modules==0.2.8
pyasn1==0.4.8
pycparser==2.20; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
pyparsing==2.4.7; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2, 3.3'
python-dateutil==2.8.2; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
pytrends==4.7.3
pytz==2021.1
requests==2.26.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4, 3.5'
rsa==4.7.2; python_version >= '3.5' and python_version < '4'
six==1.16.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
urllib3==1.26.6; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4' and python_version < '4'
